dev/sbt-plugin/src/sbt-test/sbt-plugin/projections-happy-path/hello/src/test/java/com/example/hello/impl/HelloServiceTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package com.example.hello.impl;
+
+import com.example.hello.api.HelloService;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import scala.concurrent.duration.FiniteDuration;
+
+import java.util.concurrent.TimeUnit;
+
+import static com.lightbend.lagom.javadsl.testkit.ServiceTest.*;
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.junit.Assert.assertTrue;
+
+public class HelloServiceTest {
+
+    public final Logger logger = LoggerFactory.getLogger(HelloServiceTest.class);
+
+    @Test
+    public void shouldStorePersonalizedGreeting() {
+        withServer(defaultSetup().withCassandra(), server -> {
+            HelloService service = server.client(HelloService.class);
+
+            String greetingMessage = "Hi";
+            service.useGreeting("Alice", greetingMessage).invoke().toCompletableFuture().get(5, SECONDS);
```

Comment:
I would suggest having more generous timeouts here (on this test overall, not only this line).

Reply:
The only place where more generous timeouts make sense is `eventually(new FiniteDuration(25, SECONDS)`.

Hmm, and the first hit which warms up the persistent entity. The rest are not even hitting the database.